README.md

```diff
@@ -12,13 +13,43 @@ The Application is made with:
 * Python, Django & Django-REST-Framework
 * Postgres, Tests
 
-To start run: ```python manage.py runserver```
+Set up instructions:
+
+1. Assuming you have postgres installed, create project database:
+    ```
+    sudo su - postgres  
+    psql  
+    CREATE DATABASE djangoapp;  
+    CREATE USER coder WITH PASSWORD 'coderpwd';  
+    \q  
+    exit  
+    ```
+
+2. In the folder **django-app** make a virtualenv called myenv.  
+    ```python3 -m venv myenv```
+
+3. Activate virtual environment (include full path if required).  
+    ```source djangoapp/myenv/bin/activate```
+
+4. Install requirements of the project.  
+    ```pip install -r requirements.txt```
+
+5. Create and apply migrations.  
```

Comment:
sorry to be picky, but just to avoid any possible misunderstanding I'd just write

```suggestion
5. Apply migrations.  
```